Towny/src/main/java/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -2231,7 +2231,7 @@ public boolean saveWorld(TownyWorld world) {
 		list.add("# Valid EntityTypes are listed here: https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/entity/EntityType.html");
 		list.add("isDeletingEntitiesOnUnclaim=" + world.isDeletingEntitiesOnUnclaim());
 		if (world.getUnclaimDeleteEntityTypes() != null)
-			list.add("unclaimDeleteEntityTypes=" + StringMgmt.join(world.getUnclaimDeleteEntityTypes(), ","));
+			list.add("unclaimDeleteEntityTypes=" + StringMgmt.join(world.getUnclaimDeleteEntityTypes().stream().map(type -> BukkitTools.keyAsString(type.getKey())).collect(Collectors.toList()), ","));
```

Comment:
Please extract the stream/keyAsString/collect into a new BukkitTools method that receives `world.getUnclaimDeleteEntityTypes` and the later-used `world.getPlotManagementWildRevertEntities()`, something with a good name.

Something like BukkitTools.convertEntityTypesToList()